examples/ipfs-private.rs

```diff
@@ -39,8 +39,7 @@ use libp2p::{
         either::EitherTransport, muxing::StreamMuxerBox, transport, transport::upgrade::Version,
     },
     gossipsub::{self, Gossipsub, GossipsubConfigBuilder, GossipsubEvent, MessageAuthenticity},
-    identify::{Identify, IdentifyConfig, IdentifyEvent},
-    identity,
+    identify, identity,
```

Comment:
```suggestion
    identify,
```
We only need one, right?

Reply:
Hehe, it is not the same though. `identify` vs `identity` :D

Reply:
I would lie if I said that this is the first time this mistake happened to me. :D

Sorry for the noise @thomaseizinger and @jxs.